community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,133 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
```

Comment:
What about the suggestion above or here @fgheysels ?
```suggestion
# Introducing more message flow control with Azure Service Bus in Arcus Messaging v0.5
```

Reply:
What about _Introducing improved message flow control with Azure Service Bus in Arcus Messaging_  @tomkerkhove ?

Reply:
Sounds good to me!